xAPI.md

```diff
@@ -3133,19 +3133,19 @@ Returns: ```200 OK```, Statement or [Statement Result](#retstmts) (See [Section
 			<br/><br/>
 			If "exact", return Agent, Activity, Verb and Group Objects populated exactly as they 
 			were when the Statement was received. An LRS requesting Statements for the purpose 
-			of importing them would use a format of "exact".  
+			of importing them would use a format of ```exact```.  
 			<br/><br/>
 			If "canonical", return Activity Objects and Verbs populated with the canonical
 			definition of the Activity Objects and Display of the Verbs as determined by the LRS, after
 			applying the <a href="#queryLangFiltering">language filtering process defined below</a>,
-			and return the original Agent and Group Objects as in "exact" mode.  
+			and return the original Agent and Group Objects as in ```exact``` mode.  
 		</td>
 		<td>Optional</td>
 	</tr>
 	<tr>
-		<td>attachments</td><td>Boolean</td><td>False</td>
-		<td>If true, the LRS uses the multipart response format and includes all attachments as 
-		described previously.  If false, the LRS sends the prescribed response with Content-Type 
+		<td>attachments</td><td>Boolean</td><td>false</td>
+		<td>If "true", the LRS uses the multipart response format and includes all attachments as 
```

Comment:
Hmm, using double quoted strings might lead some people to try putting strings there. Maybe code-formatting instead of quoting? Same below for the other two occurrences.